main/src/main/scala/sbt/Defaults.scala

```diff
@@ -1424,12 +1432,16 @@ object Defaults extends BuildCommon {
   def pickMainClass(classes: Seq[String]): Option[String] =
     sbt.SelectMainClass(None, classes)
 
-  private def pickMainClassOrWarn(classes: Seq[String], logger: Logger): Option[String] = {
+  private def pickMainClassOrWarn(
+      classes: Seq[String],
+      logger: Logger,
+      logWarning: Boolean
+  ): Option[String] = {
     classes match {
-      case multiple if multiple.size > 1 =>
-        logger.warn(
-          "Multiple main classes detected.  Run 'show discoveredMainClasses' to see the list"
-        )
+      case multiple if multiple.size > 1 && logWarning =>
+        val msg =
+          "Multiple main classes detected. Run 'show discoveredMainClasses' to see the list."
```

Comment:
```suggestion
          "multiple main classes detected: run 'show discoveredMainClasses' to see the list"
```

This is purely cosmetic but I'd like to unify towards lowercase for warnings (already the case in most Scala compiler errors etc)

Reply:
Sure.